### PR TITLE
Friendly URL for component schemas

### PR DESCRIPTION
**Pull request by gabog**
Relates to #5840

- Created v{version} folder with copies of current schemas that matches versioning schema defined by @clearab  for skills.
- Updated readme to note new location and legacy versions

### CODE REVIEW

File: schemas/component/readme.md [hunk 1]
```diff
@@ -5,3 +5,19 @@ Bot Framework declarative component definitions (via .schema files which describ
...
+## Published location
+
+The component JSON schemas are published at:
+
+`https://schemas.botframework.com/schemas/component/v{version}/{schema file}`
+
+Examples:
+
+`https://schemas.botframework.com/schemas/component/v1.0/component.schema`
+`https://schemas.botframework.com/schemas/component/v1.0/definitions.schema`
```
Comment (chrimc62) [inline schemas/component/readme.md:17]: This is obsolete and can be removed.  There is only component.schema--it includes everything in definitions.schema.  That was a point in time thing.

Comment (gabog) [reply schemas/component/readme.md]: Cool, I updated the readme and deleted definitions.schema from the v1.0 folder. Makes sense?

Comment (gabog) [reply schemas/component/readme.md]: Oh, and yes, I am planning to go and update the SDK schema files once this PR is approved and merged (the friendly URL only maps to the master branch and I need this merged first)

**Comment by vishwacsena** [comment]
@gabog can you pick up the changes in my PR into this as well? If not, I can issue one after your is merged.

https://github.com/microsoft/botframework-sdk/pull/5954

**Pull request merged**